.github/workflows/PR.yml

```diff
@@ -21,6 +22,4 @@ jobs:
     - name: Install Tox and any other packages
       run: pip install tox
     - name: Run Tox
-      env: 
-        RAPID_API_TOKEN: ${{secrets.RAPID_API_TOKEN}}
```

Comment:
Just out of curiosity, why did you remove RAPID_API_TOKEN from PR.yaml? Won't we need it for checks?

Reply:
We don't need it yet so I removed it, we can add it once we need it